Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: /app/samvera/hyrax-webapp
   specs:
-    iiif_print (3.0.6)
+    iiif_print (3.0.7)
```

Comment:
I'm surprised to see changes here from a version update 🤔 It may be ok but curious how this got updated

Reply:
I update the version, bundle, and save Gemfile.lock. This way it reflects the new version.